### PR TITLE
Add more Appointment Tests

### PR DESCRIPTION
**Pull request by ReubenTeng**

### CODE REVIEW

File: src/main/java/seedu/address/model/appointment/UniqueAppointmentList.java [hunk 1]
```diff
@@ -50,13 +50,13 @@ public void add(Appointment toAdd) {
      */
     public void setAppointment(Appointment target, Appointment editedAppointment) {
         requireAllNonNull(target, editedAppointment);
-
         int index = internalList.indexOf(target);
         if (index == -1) {
+            System.out.println("Here");
```
Comment (kaitlynng) [inline src/main/java/seedu/address/model/appointment/UniqueAppointmentList.java]: Remove testing print lines

File: src/test/java/seedu/address/model/appointment/UniqueAppointmentListTest.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
+package seedu.address.model.appointment;
```
Comment (kaitlynng) [inline src/test/java/seedu/address/model/appointment/UniqueAppointmentListTest.java]: Changed implementation of UniqueAppointmentList to UniqueList (generic interface)

File: src/test/java/seedu/address/testutil/AppointmentBuilder.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import seedu.address.model.tag.Tag;
+import seedu.address.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building Appointment objects.
+ */
+public class AppointmentBuilder {
+
+    public static final Patient DEFAULT_PATIENT = ALICE;
+    public static final String DEFAULT_DATETIME = "2020-12-12T23:59:59";
```
Comment (kaitlynng) [inline src/test/java/seedu/address/testutil/AppointmentBuilder.java:22]: AppointmentBuilder fields need to change to match new implementation of Appointment

**Review by kaitlynng — APPROVED** [review]
Looks good! Need to change the tests for Appointment class (to use the new DateTime class in common) and UniqueAppointmentList (now using a UniqueList generic interface)

**Review by jianhandev — APPROVED** [review]

**Pull request merged**